bigtable/admin.go

```diff
@@ -1584,14 +1584,21 @@ func UpdateInstanceAndSyncClusters(ctx context.Context, iac *InstanceAdminClient
 
 // RestoreTable creates a table from a backup. The table will be created in the same cluster as the backup.
 func (ac *AdminClient) RestoreTable(ctx context.Context, table, cluster, backup string) error {
-	ctx = mergeOutgoingMetadata(ctx, ac.md)
-	prefix := ac.instancePrefix()
-	backupPath := ac.backupPath(cluster, backup)
+	return ac.RestoreTableFrom(ctx, ac.instance, cluster, table, backup)
+}
 
+// RestoreTableFrom creates a new table in the admin's instance by restoring from the specified backup in a different instance.
```

Comment:
Does it have to be a different instance? Could it be the same instance? Should people prefer this or `RestoreTable`? Should we update this comment to refer to `RestoreTable` or `RestoreTable`'s comment to refer to this?

Reply:
Done, thanks for the suggestion